docker/Dockerfile-workers

```diff
@@ -14,9 +14,6 @@ COPY ./docker/conf-workers/* /conf/
 # Expose nginx listener port
 EXPOSE 8080/tcp
 
-# Volume for user-editable config files, logs etc.
-VOLUME ["/data"]
```

Comment:
This is just a driveby while Im in the area, for the same reason as https://github.com/matrix-org/synapse/pull/11997.